Add Outdated k6 Release Used to Performance Insights (#755)
grafana/k6-docs: src/data/markdown/docs/03 cloud/02 Analyzing Results/02 Performance Insights.md

</CodeGroup>

### Outdated k6 release used

*Identifier*: `best_practice_outdated_k6_release_used`
- **Happens when**:
  you use a legacy version of k6 that is significantly older than the latest stable version.
- **Recommendations**:

>   - Download and install the latest release of k6 [here](/getting-started/installation) or upgrade your packages if relevant.

Reviewer: ```suggestion
  - [Install the latest release of k6](/getting-started/installation), or upgrade your existing packages.
```

>   - Update the k6 binary that your CI/CD pipeline uses for running tests.

Reviewer: ```suggestion
  - Update the k6 binary that your CI/CD pipeline uses to run tests.
```

</CodeGroup>

### Outdated k6 release used

*Identifier*: `best_practice_outdated_k6_release_used`
- **Happens when**:
  you use a legacy version of k6 that is significantly older than the latest stable version.
- **Recommendations**:
  - Download and install the latest release of k6 [here](/getting-started/installation) or upgrade your packages if relevant.
  - Update the k6 binary that your CI/CD pipeline uses for running tests.
  - If you're part of an organization or team, collectively decide on a version of k6 to use going forward for consistency and ease of comparison.

>   - If you have questions about which version of k6 to use, please contact our [Support team](mailto:support@k6.io).

Reviewer 2: I think this line is more for in-app support
